KinWallet/View Controllers and Co./More/HelpCenterViewController.swift

```diff
@@ -65,6 +71,45 @@ final class HelpCenterViewController: WebViewController {
     fileprivate func loadMainPage() {
         loadURL(helpCenterURL)
     }
+
+    fileprivate func presentErrorAlert() {
+        var title = L10n.SupportSubmitError.title
```

Comment:
In general, whenever `let` can be used, it is recommended over `var`.
There are two ways it can be done here:

1 - You can declare `let title: String` `let message: String` and set the values inside the if and else
2 - You can use the ternary operator: `let title = errorCount > 0 ? L10n.SupportSubmitError.AfterRetry.title : L10n.SupportSubmitError.title `